api/api.go

```diff
@@ -45,29 +50,39 @@ type Configer interface {
 
 //======= Interface for module 'network' =======
 
+// ConnectionHandler handles incoming connections.
+//
+// PeerMessageStreamHandler method provides a mechanism to initiate
+// message exchange with the specified peer replica. It must only be
```

Comment:
Replica is a core concept of BFT algorithm, but sometimes seem to be called like "peer" or "peer replica". Do we call them to represent exactly the same things, or use them differently for the situations?

Reply:
Although, I'm not sure if that is a correct way, I tend to use "replica" when referring to _any_ replica, and "peer replica" (or shortly "peer") when referring to a replicas _other_ than the current one. In different words, "replica" is used from the external or client's perspective, whereas "peer" from a replica's point of view.

@jcs47 What do you think?